modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -0,0 +1,151 @@
+Consider a secondary workload CR, which is not compliant with the PodSpec, with containers at the `spec.containers` path:
+
+.Example: Secondary workload CR
+[source,yaml]
+----
+apiVersion: "operator.sbo.com/v1"
+kind: SecondaryWorkload
+metadata:
+    name: secondary-workload
+spec:
+    containers:
+    - name: hello-world
+      image: quay.io/baijum/secondary-workload:latest
+      ports:
+      - containerPort: 9966
+----
+
+The following example shows how to configure the value of where the `spec.containers` path is by specifying it in the `ServiceBinding` CR and bind this path at a `spec.application.bindingPath.containersPath` custom location:
+
+.Example: `ServiceBinding` CR with the `spec.containers` path in a custom location
+[source,yaml]
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+spec:
+    services:
+    - group: postgres-operator.crunchydata.com
+      version: v1beta1
+      kind: PostgresCluster
+      name: hippo
+      id: postgresDB
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-pguser-hippo
+      id: postgresSecret
+    application: <1>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    application: <2>
+      name: secondary-workload
+      group: operator.sbo.com
+      version: v1
+      resource: secondaryworkloads
+      bindingPath:
+        containersPath: spec.containers <3>
+----
+<1> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
+<2> Secondary workload, which is not compliant with the PodSpec.
+<3> Custom location of the container path.
+
+After you specify the location of the container path, {servicebinding-title} generates the binding data, which becomes available in the container path specified in the secondary workload of the `ServiceBinding` CR.
+
+The following example shows the `spec.containers` path, which is updated  with the `envFrom` and `secretRef` fields:
+
+.Example: Secondary workload CR with the `envFrom` and `secretRef` fields
+[source,yaml]
+----
+apiVersion: "operator.sbo.com/v1"
+kind: SecondaryWorkload
+metadata:
+    name: secondary-workload
+spec:
+    containers:
+    - env: <1>
+      - name: ServiceBindingOperatorChangeTriggerEnvVar
+        value: "31793"
+      envFrom:
+      - secretRef:
+          name: secret-resource-name <2>
+      image: quay.io/baijum/secondary-workload:latest
+      name: hello-world
+      ports:
+      - containerPort: 9966
+      resources: {}
+----
+<1> Unique array of containers with values generated by the {servicebinding-title}. These values are based on the backing service CR.
+<2> Name of the `Secret` resource generated by the {servicebinding-title}.
+
+=== Configuring the custom location of the secret path
+Consider a secondary workload CR, which is not compliant with the PodSpec, with only the secret at the `spec.secret` path:
```

Comment:
```suggestion
Consider a secondary workload CR that is not compliant with the PodSpec with only the secret at the `spec.secret` path:
```
Again, that relative clause confuses me. It makes it sound like all secondary workload CRs are not compliant.

Reply:
Yes, it is correct. so ignoring your revision here.